### PR TITLE
Provide more info when CI fails

### PR DESCRIPTION
**Pull request by Marenz**
refs #5803

### CODE REVIEW

File: test/RPCSession.cpp [hunk 1]
```diff
@@ -166,7 +166,6 @@ RPCSession::TransactionReceipt RPCSession::eth_getTransactionReceipt(string cons
 {
 	TransactionReceipt receipt;
 	Json::Value const result = rpcCall("eth_getTransactionReceipt", { quote(_transactionHash) });
-	BOOST_REQUIRE(!result.isNull());
```
Comment (chriseth) [inline test/RPCSession.cpp:169]: Shouldn't we at least `BOOST_CHECK`?

Comment (Marenz) [reply test/RPCSession.cpp:169]: We do the same check inside the `rpcCall` function now

Comment (Marenz) [reply test/RPCSession.cpp:169]: though... maybe we aren't actually. Maybe it can be member but be `null` too. I should be more precise in my if check

**Comment by Marenz** [comment]
Updated, now also checking for null again

**Comment by Marenz** [comment]
My motivation is that we get more information next time the CI randomly fails

**Review by chriseth — APPROVED** [review]

**Review by axic — APPROVED** [review]
Looks good as it checks for both null-ness and presence.

**Pull request merged**